src/main/java/com/github/aloomaio/androidsdk/util/HttpService.java

```diff
@@ -88,7 +95,14 @@ public boolean isOnline(Context context) {
                 connection = (HttpURLConnection) url.openConnection();
                 connection.setConnectTimeout(2000);
                 connection.setReadTimeout(10000);
-                if (null != params) {
+
+                if (headers != null){
+                    for (Map.Entry<String, String> header: headers.entrySet()){
+                        connection.setRequestProperty(header.getKey(), header.getValue());
```

Comment:
there is also a `connection.addRequestProperty` but that one does not overwrite existing values associated with the same key. So I think `setRequestProperty` is safer for our purposes